migrations/sqls/20190806211241-client-interaction-up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS clientInteractions (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    clientInteractionType ENUM (
+        'clientInteractionCreated',
+        'clientInteractionUpdated',
+        'clientInteractionDeleted'
+    ) NOT NULL,
+    clientId INT NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    serviceId INT NOT NULL,
+    interactionType ENUM (
+        'in pesron',
+        'by phone',
+        'workshop/talk',
+        'one on one/light touch',
+        'consultation'
+    ) NOT NULL,
+    description MEDIUMTEXT,
```

Comment:
This column should be removed, for the same reason as described in my comment about the `title` column